KlockanAPI/src/KlockanAPI.Infrastructure/CrossCutting/Authentication/KeyCloakAuthentication.cs

```diff
@@ -27,27 +27,28 @@ public static AuthenticationBuilder AddKeyCloakJWTAuthentication(this IServiceCo
             var certs = await GetCerts(KeyCloakSecrets!["CertsUrl"]!);
 
             // Obtener los certificados
-            if(certs != null)
+            if (certs != null)
             {
                 // Obtener el token actual
                 o.Events = new JwtBearerEvents
                 {
                     OnTokenValidated = context =>
                     {
                         var token = context.SecurityToken as JwtSecurityToken;
-                        if(token != null)
+                        if (token != null)
                         {
                             var jit = token.Payload["jit"] as string;
 
-                            foreach(var cert in certs)
+                            foreach (var cert in certs)
                             {
                                 var jwk = new JsonWebKey(cert);
                                 var validationParameters = new TokenValidationParameters
                                 {
                                     ValidateIssuerSigningKey = true,
                                     IssuerSigningKey = jwk,
                                     ValidateIssuer = true,
-                                    ValidateAudience = false
+                                    ValidateAudience = false,
+                                    // RoleClaimType = "roles"
```

Comment:
This comment is necessary to be used in the future? or can we delete it?

Reply:
fixed it